cloudhub-2/modules/ROOT/pages/ps-create-configure-tgw.adoc

```diff
@@ -166,6 +166,51 @@ To test the connection, use the Network Tools application.
 
 For download and usage information about the Network Tools application, see https://help.mulesoft.com/s/article/How-To-Use-Network-Tools-Application[How To Use Network Tools Application^] (KB article).
 
+[[best-practices]]
+== Maintain the Transit Gateway 
+
+After you test the connection to your private space, periodically ensure your transit gateway connection is maintained. 
+
+If your private space loses access to the transit gateway, the state of the transit gateway is changed to `unknown` or `deleted`. If the transit gateway remains in the `unknown` or `deleted` state for 14 days, Anypoint Platform may remove the private space attachments from the transit gateway. However, the transit gateway itself is not removed. 
+
+Your private space loses access to a transit gateway when:
+
+* The transit gateway is deleted from CloudHub 2.0.
+* The transit gateway is deleted from the AWS console.
+* A resource share is deleted.
+* The MuleSoft account principal is deleted from the resource share. 
+
+If you want to remove or delete the transit gateway from CloudHub 2.0, you should complete the steps in <<remove-tgw>> or <<delete-tgw>> before deleting the transit gateway, transit gateway attachment, or resource share from the AWS console. Removing connections from the AWS console without removing them from CloudHub 2.0 disrupts the connection but does not remove it. 
+
+[[troubleshoot-tgw]]
+== Troubleshoot States for the Transit Gateway or Transit Gateway Attachment
+
+When the transit gateway connection is disrupted, the state of the transit gateway or transit gateway attachment changes to `unknown`, `rejected`, or `deleted`. Depending on the cause of the state change, you can fix the state of the transit gateway or transit gateway attachment by deleting or detaching them.
```

Comment:
Would this fix also include re-adding or reattaching?

Reply:
Or adding the principal?

Reply:
I will check.

Reply:
```suggestion
When the transit gateway connection is disrupted, the state of the transit gateway or transit gateway attachment changes to `unknown`, `rejected`, or `deleted`. Depending on the cause of the state change, you can fix the state of the transit gateway or transit gateway attachment by deleting and recreating or detaching and reattaching them.
```